tron/kubernetes.py

```diff
@@ -433,7 +437,23 @@ def recover(self, task: KubernetesTask) -> None:
         """
         Given an instance of a KubernetesTask, attempt to reconcile the current state of the task from Kubernetes.
         """
-        pass
+        if not task:
+            return
+
+        if not self.enabled:
+            task.log.info("Could not recover task, Kubernetes usage is disabled.")
+            task.exited(None)
+            return
+
+        self._check_connection()
+
+        # pod name
+        task_id = task.get_kubernetes_id()
+        self.tasks[task_id] = task
+        task.log.info("TRON RESTARTED! Starting recovery procedure by reconciling state for this task from Kubernetes")
+        task.started()
+        self.runner.reconcile(task.get_config())  # type: ignore  # we need to add type annotation to task_proc
```

Comment:
Ah interesting, I saw this on `kill` and assumed it would also apply to mine.

The error I was getting was:
```
py36 run-test: commands[6] | mypy --pretty tron/kubernetes.py
tron/kubernetes.py: note: In member "recover" of class "KubernetesCluster":
tron/kubernetes.py:455:9: error: Item "None" of "Optional[Subscription]" has no attribute "reconcile"
            self.runner.reconcile(task.get_config())
```
But this is just because I didnt' assert that self.runner was not None before calling reconcile here, as we do in submit(task). I'll go that route instead.